_rules/autocomplete-valid-value-73f2c2.md

```diff
@@ -62,6 +59,8 @@ The `aria-disabled` state is used on `input` elements which are not part of [seq
 
 The intent of this rule is to ensure that the `autocomplete` attribute can be used to support personalization. Many users may find it easier to fill out forms if those can be styled or laid out in a way that is familiar to them. Assistive technologies can do this when a form control is marked up in such a way that its purpose can be understood. For instance, assistive technologies could add familiar icons and colors to different fields, making it easier for the user to understand what the form does.
 
+Many browsers provide auto-filling suggestions even when the control's `type` [attribute value][] is not appropriate for its `autocomplete` [attribute value][]. The same happens when the `autocomplete` property is queried. However, the `autocomplete` property is not programmatically identifiable if the requirements for the optional tokens are not met.
```

Comment:
```suggestion
Many browsers provide auto-filling suggestions even when the control's `type` [attribute value][] is not [appropriate][] for its `autocomplete` [attribute value][]. The same happens when the `autocomplete` property is queried. However, the `autocomplete` property is not programmatically identifiable if the requirements for the optional tokens are not met.
```
I'd keep the link to the specs here since it has a very specific meaning and it is not obvious what it means.